packages/styled/package.json

```diff
@@ -49,20 +49,70 @@
   },
   "exports": {
```

Comment:
Just above this we have `package.json#browser`, we probably should just remove that from all packages. But then if we meant to support older browser we really should have the default bundle to contain the runtime check. The proposed setup wont benefit older bundlers but it wont break them.

Reply:
Yeah, definitely think the default bundle should include the runtime check like the current one does

Reply:
ok, I'm removing `package.json#browser` since with `importConditions` those entries are not even auto-updated anymore